src/CMakeLists.txt

```diff
@@ -133,11 +133,16 @@ execute_process(
 )
 string(REGEX REPLACE "[\r\n]" "" BUILD_DATE "${BUILD_DATE}")
 
-# Set the requirement for the C++ standard
-set(CMAKE_CXX_STANDARD 11)
```

Comment:
Wouldn't it be easier to put the `11` into and option and set it to `14` with MacPorts?